openapi.yaml

```
@@ -6037,7 +6037,14 @@ components:
           $ref: '#/components/schemas/log_level'
         message:
           type: string
-          description: A message explaining the log entry.
+          description: >-
+            A concise message explaining the log entry.
+
+            Messages do *not* explicitly support [CommonMark 0.29](http://commonmark.org/)
+            syntax as other descriptive fields in the openEO API do,
+            but the messages may contain line breaks or indentation.
```

suggestion:

> ... MAY may contain line breaks or indentation, to be rendered as such.

---

```suggestion
            but the messages MAY contain line breaks or indentation.
```

I don't think we should add a requirement here that client should render them as such. We don't require formatting related capabilities anywhere else, I think.